lms/djangoapps/ora_staff_grader/mock/data/ora_metadata.json

```diff
@@ -0,0 +1,80 @@
+{
+    "block-a": {
+        "name": "Individual ORA",
+        "prompts": [
+            {
+                "description": "<p>Enter a text/files response.</p>"
+            }
+        ],
+        "type": "individual",
+        "rubricConfig": {
+            "feedback": "optional",
```

Comment:
What is this? I see it in rubricConfig don't believe this is in the actual feedback config from the backend and I don't see us inserting this anywhere.

Reply:
I assumed this was how we tell whether the "overall feedback" is required or optional. Doesn't that _have_ to exist somewhere?